pytradfri/command.py

```diff
@@ -26,52 +31,52 @@ def __init__(
         self._err_callback = err_callback
         self._observe = observe
         self._observe_duration = observe_duration
-        self._raw_result = None
-        self._result = None
+        self._raw_result: Optional[str] = None
+        self._result: Optional[str] = None
 
     @property
-    def method(self):
+    def method(self) -> str:
         return self._method
 
     @property
-    def path(self):
+    def path(self) -> list[str]:
         return self._path
 
     @property
-    def data(self):
+    def data(self) -> Optional[dict[str, Any]]:
```

Comment:
You get the idea, there're few more places.
```suggestion
    def data(self) -> dict[str, Any] | None:
```